File: backend/Models/Station.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var StationSchema = new mongoose.Schema(
{
  Name:             { type: String, required: true, index: { unique: true } },
  Grade:            { type: Number, required: true },
  Subject:          { type: String, required: true },
  Description:      { type: String, required: true },
  User:             { type: Schema.ObjectId, required: false },
  Tdot:             { type: Schema.ObjectId, required: false },
});

module.exports = mongoose.model('Station', StationSchema);

File: mobileApp/uGuide/Data/Models/Station.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


namespace uGuide.Data
{
    public class Station
    {
        public string name { get; set; }
        public int grade { get; set; }
        public string subject { get; set; }
        public string description { get; set; }

        public Station(string name, int grade, string subject, string description)
        {
            this.name = name;
            this.grade = grade;
            this.subject = subject;
            this.description = description;
        }
    }
}